assets/i18n/en_US.json

```diff
@@ -163,13 +163,15 @@
     },
     "installerView": {
         "widgetTitle": "Installer",
-        "installType": "Select install type",
-        "installTypeDescription": "Select the installation type to proceed with.",
+        "installType": "Choose installation method",
+        "installTypeDescription": "To install, you can either mount or install normally. Mounting will install the patched app on top of an existing installation.",
 
         "installButton": "Install",
         "installRootType": "Mount",
         "installNonRootType": "Normal",
 
+        "warning": "Disable auto updates after installing the app to avoid unexpected issues.",
```

Comment:
Is this in scope of this PR?

Reply:
Yes, it is necessary to let the user know what precautions to take after the installation